src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -294,6 +294,16 @@
       <Context>ProjectSubscriptionService;BrowseObject</Context>
     </PropertyPageSchema>
 
+    <!-- Framework references -->
+    <PropertyPageSchema Include="$(ManagedXamlNeutralResourcesDirectory)FrameworkReferenceProjectBuild.xaml">
```

Comment:
Note - these are very deliberately named "ProjectBuild" to indicate that they come from build, as opposed to evaluation, similar to the "ResolvedXXX" concept. Open to a better name, maybe "CollectedFrameworkReference" to match the "CollectXXX" targets that are called to return them?

I want a naming convention as I was extremely surprised that `PackageReference` appears to be an evaluation item but actually causes a design-time build.

Reply:
I renamed these to "CollectedXXX"